be/src/vec/exprs/vstruct_literal.cpp

```diff
@@ -25,8 +25,7 @@ Status VStructLiteral::prepare(RuntimeState* state, const RowDescriptor& row_des
     Field struct_field = Tuple();
     for (const auto child : _children) {
         Field item;
-        ColumnPtrWrapper* const_col_wrapper = nullptr;
-        RETURN_IF_ERROR(child->get_const_col(context, &const_col_wrapper));
```

Comment:
if remove the  RETURN_IF_ERROR，no need to report the error message ?

Reply:
Thanks. It is a mistake.